src/main/scala/it/pagopa/interop/agreementprocess/common/system/ApplicationConfiguration.scala

```diff
@@ -32,7 +32,7 @@ object ApplicationConfiguration {
 
     ReadModelConfig(connectionString, dbName)
   }
-
-  val certifiedMailQueueName: String = config.getString("agreement-process.certified-mail-queue-name")
+  val eventsQueueName: String          = config.getString("agreement-process.events-queue-name")
```

Comment:
[same](https://github.com/pagopa/interop-be-agreement-process/pull/238/files#r1253046683) considarations